src/Npgsql/PostgresDatabaseInfo.cs

```diff
@@ -142,10 +145,10 @@ static string GenerateLoadTypesQuery(bool withRange, bool withMultirange, bool l
 WHERE
     typtype IN ('b', 'r', 'm', 'e', 'd') OR -- Base, range, multirange, enum, domain
     (typtype = 'c' AND {(loadTableComposites ? "ns.nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast')" : "relkind='c'")}) OR -- User-defined free-standing composites (not table composites) by default
-    (typtype = 'p' AND typname IN ('record', 'void')) OR -- Some special supported pseudo-types
+    (typtype = 'p' AND typname IN ('record', 'void', 'unknown')) OR -- Some special supported pseudo-types
```

Comment:
Is there a specific reason we need to handle "unknown"?

Reply:
Not really, we query for it in PgSerializerOptions, but we could indeed use `UnknownBackendType.Instance` as well. I don't mind having some of these important pseudo types being truly present in the catalog though.

Reply:
I don't mind either, was just curious about this change.

Reply:
Ah and we also query for it during writing. We must find it in the catalog to be able to resolve its PgTypeId.